lib/eventsource.js

```diff
@@ -167,22 +164,20 @@ function EventSource (url, eventSourceInitDict) {
       res.on('close', function () {
         res.removeAllListeners('close')
         res.removeAllListeners('end')
-        onConnectionClosed()
+        //onConnectionClosed()
```

Comment:
We shouldn't leave lines commented out, but why remove this? We need to call this when we close, no?

Reply:
It's completely my fault on the removed 302 and the commented `onConnectionClosed()` Both are leftover from my testing and shouldn't be like that. I'm still not sure of what I'm doing with pull requests and it seems that I might have merged the wrong commit, sorry... :/